core/storage/statistics/gae_models.py

```diff
@@ -1226,12 +1226,16 @@ class StateAnswersCalcOutputModel(base_models.BaseMapReduceBatchResultsModel):
     exploration_version = ndb.StringProperty(indexed=True, required=True)
     state_name = ndb.StringProperty(indexed=True, required=True)
     calculation_id = ndb.StringProperty(indexed=True, required=True)
+    # Calculation output type (for deserialization). See
+    # stats_domain.StateAnswersCalcOutput for an enumeration of valid types.
+    calculation_output_type = ndb.StringProperty(indexed=True)
```

Comment:
Will the introduction of this new field break existing models in production?

Reply:
It shouldn't since this isn't a required field, but we should reset StateAnswersCalcOutputModel as part of the next release regardless since it will break new code.